src/account/operations/transaction/high_level/sweep_outputs.rs

```diff
@@ -100,95 +97,92 @@ impl AccountHandle {
-            //             // account without unspent outputs can't be related to this output
-            //             &addresses_with_unspent_outputs,
-            //             &[Address::Nft(NftAddress::new(nft_id))],
-            //             &output_data,
-            //             current_time,
-            //         ) {
-            //             outputs_to_sweep.push(output_data);
-            //         } else {
-            //     // do we care about them?
-            //             other_owned_outputs.push(output_data);
-            //         }
-            //     } else {
-            //         outputs_to_sweep.push(output_data);
-            //     }
-            // }
-
-            let mut output_ids = Vec::new();
-            let mut outputs = Vec::new();
-
-            let network_id = self.client.get_network_id().await?;
-
-            for mut output_response in basic_outputs {
-                if output_response.metadata.is_spent {
-                    continue;
-                }
+    ) -> crate::Result<Transaction> {
+        let token_supply = self.client.get_token_supply().await?;
+        let bech32_hrp = self.client().get_bech32_hrp().await?;
+        let address = AddressWrapper::new(address, bech32_hrp);
+        let remainder_address = self.get_sweep_remainder_address(&address, options).await?;
+
+        // TODO: get outputs from unspent outputs
+        let basic_outputs: Vec<OutputWithMetadataResponse> = Vec::new();
+        // maybe something like below
+        // let mut outputs_to_sweep = Vec::new();
+        // let mut other_owned_outputs = Vec::new();
+
+        // for output_data in self.list_unspent_outputs(None).await? {
+        //     // Ignore outputs with a single [UnlockCondition], because then it's an
+        //     // [AddressUnlockCondition] and we own it already without
+        //     // further restrictions
+        //     if output_data
+        //         .output
+        //         .unlock_conditions()
+        //         .expect("output needs to have unlock_conditions")
+        //         .len()
+        //         != 1
+        //     {
+        //         if can_output_be_unlocked_now(
+        //             // We use the addresses with unspent outputs, because other addresses of the
+        //             // account without unspent outputs can't be related to this output
+        //             &addresses_with_unspent_outputs,
+        //             &[Address::Nft(NftAddress::new(nft_id))],
+        //             &output_data,
+        //             current_time,
+        //         ) {
+        //             outputs_to_sweep.push(output_data);
+        //         } else {
+        //     // do we care about them?
+        //             other_owned_outputs.push(output_data);
+        //         }
+        //     } else {
+        //         outputs_to_sweep.push(output_data);
+        //     }
+        // }
```

Comment:
What's up with that?

Reply:
Not sure, its in our code: https://github.com/iotaledger/wallet.rs/blob/develop/src/account/operations/transaction/high_level/sweep_outputs.rs#L116